### ISSUE TITLE
Syntax highlight with vue-prism

### ISSUE THREAD

**Issue by albinotonnina** [sdras/object-explorer#17]
I see that the dep is there already but prism is not fully implemented.
I tried to play with it a few minutes, adding the missing `prism.css` and changing the markup but it wasn't coming out well. Maybe someone can take a look at it.
I think there's something also with the format of the code coming from the store.

Also, do you think is good to keep strings and code together in the store like that and have the translation files basically duplicating maybe unnecessarily the code?
Could a next iteration maybe address differently the thing? Like having the code in a different format?

**Comment by sdras** [comment]
I had the syntax highlighting working, but took it out because there is no current support for the `pre` tag in splittext (I use this for the animation). I left it in because I was hoping to tackle it at some point, but as it's a small amount of code, it wasn't crucial to the project. If anyone wants to dig into that, I'm happy to have the help as I'm quite busy right now. It's not vital though. If you want to see how it looks set up, it's in the history in the array-explorer repo.

You are correct in assuming that the store files would no longer need the `<br>` and `<span>` tags if this was fixed.

**Comment by albinotonnina** [comment]
Yes yes, nothing really vital. I may try in my free time as well or someone else at some point.

**Comment by sdras** [comment]
Please feel free to :) thanks for the inquiry
